core/src/main/kotlin/org/evomaster/core/problem/graphql/IntrospectiveQuery.kt

```diff
@@ -70,6 +83,22 @@ class IntrospectiveQuery {
          */
         val body = response.readEntity(String::class.java)
 
+        //TODO parse this body here to see if it has any "errors" field and no "data"
+
+        val jackson = ObjectMapper()
+
+        val node: JsonNode = try {
+            jackson.readTree(body)
+        } catch (e: JsonProcessingException) {
+            return null
```

Comment:
this should not be ignored.
if what returned is not a JSON, should log it as an error and throw a `SutProblemException` with a meaningful error message